subschedule_test.go

```diff
@@ -0,0 +1,40 @@
+package stripe
+
+import (
+	"encoding/json"
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+	"github.com/stripe/stripe-go/form"
+)
+
+func TestSubscriptionScheduleParams_AppendTo(t *testing.T) {
+	{
+		params := &SubscriptionScheduleParams{StartDateNow: Bool(true)}
+		body := &form.Values{}
+		form.AppendTo(body, params)
+		t.Logf("body = %+v", body)
+		assert.Equal(t, []string{"now"}, body.Get("start_date"))
+	}
+}
+
+func TestSubscriptionSchedule_UnmarshalJSON(t *testing.T) {
+	// Unmarshals from a JSON string
+	{
+		var v SubscriptionSchedule
+		err := json.Unmarshal([]byte(`"sub_sched_123"`), &v)
```

Comment:
Thanks for making even the token prefixes accurate. I had to look this one up to make sure :)